include/openssl/x509v3.h.in

```diff
@@ -912,6 +912,7 @@ int X509v3_addr_add_range(IPAddrBlocks *addr,
                           const unsigned afi, const unsigned *safi,
                           unsigned char *min, unsigned char *max);
 unsigned X509v3_addr_get_afi(const IPAddressFamily *f);
+unsigned X509v3_addr_get_safi(const IPAddressFamily *f);
```

Comment:
All newly added public API functions *must* come with documentation. Please can you add something? Bonus points if you also document `X509v3_addr_get_afi` which is missing documentation (or indeed any other `X509v3_addr_*` function - since they're all missing documentation).